vendors/pc/boards/windows/aws_demos/config_files/aws_demo_config.h

```diff
@@ -65,4 +65,16 @@
 /* Send AWS IoT MQTT traffic encrypted to destination port 443. */
 #define democonfigMQTT_AGENT_CONNECT_FLAGS             ( mqttagentREQUIRE_TLS | mqttagentUSE_AWS_IOT_ALPN_443 )
 
+#define democonfigMEMORY_ANALYSIS
+
+#ifdef democonfigMEMORY_ANALYSIS
+    #define demoMEMORY_ANALYSIS_STACK_DEPTH_TYPE    UBaseType_t
+    #define demoMEMORY_ANALYSIS_MIN_EVER_HEAP_SIZE( x )     xPortGetMinimumEverFreeHeapSize()
```

Comment:
Note: Newer versions of FreeRTOS have a different heap min size API that does take a parameter.

Reply:
hmm, why we are taking parameter in macro definition? ```xPortGetMinimumEverFreeHeapSize()``` doesn't require anything input.